layers/stateless/sl_ray_tracing.cpp

```diff
@@ -913,40 +915,33 @@ bool StatelessValidation::ValidateAccelerationStructureBuildGeometryInfoKHR(
                 const Location geometry_loc = info_loc.dot(Field::pGeometries, j);
-                skip |= ValidateRangedEnum(
-                    loc, ParameterName("pInfos[%i].pGeometries[%i].geometryType", ParameterName::IndexVector{i, j}),
-                    "VkGeometryTypeKHR", pInfos[i].pGeometries[j].geometryType,
-                    "VUID-VkAccelerationStructureGeometryKHR-geometryType-parameter");
+                skip |= ValidateRangedEnum(geometry_loc.dot(Field::geometryType), "VkGeometryTypeKHR",
+                                           pInfos[i].pGeometries[j].geometryType,
+                                           "VUID-VkAccelerationStructureGeometryKHR-geometryType-parameter");
                 if (pInfos[i].pGeometries[j].geometryType == VK_GEOMETRY_TYPE_TRIANGLES_KHR) {
                     constexpr std::array allowed_structs = {
                         VK_STRUCTURE_TYPE_ACCELERATION_STRUCTURE_TRIANGLES_OPACITY_MICROMAP_EXT};
 
-                    skip |= ValidateStructType(
-                        loc, ParameterName("pInfos[%i].pGeometries[%i].geometry.triangles", ParameterName::IndexVector{i, j}),
-                        "VK_STRUCTURE_TYPE_ACCELERATION_STRUCTURE_GEOMETRY_TRIANGLES_DATA_KHR",
-                        &(pInfos[i].pGeometries[j].geometry.triangles),
-                        VK_STRUCTURE_TYPE_ACCELERATION_STRUCTURE_GEOMETRY_TRIANGLES_DATA_KHR, false, kVUIDUndefined,
-                        "VUID-VkAccelerationStructureGeometryTrianglesDataKHR-sType-sType");
-                    skip |= ValidateStructPnext(
-                        loc, ParameterName("pInfos[%i].pGeometries[%i].geometry.triangles.pNext", ParameterName::IndexVector{i, j}),
-                        NULL, pInfos[i].pGeometries[j].geometry.triangles.pNext, allowed_structs.size(), allowed_structs.data(),
-                        GeneratedVulkanHeaderVersion, "VUID-VkAccelerationStructureGeometryTrianglesDataKHR-pNext-pNext",
-                        kVUIDUndefined);
-                    skip |= ValidateRangedEnum(loc,
-                                               ParameterName("pInfos[%i].pGeometries[%i].geometry.triangles.vertexFormat",
-                                                             ParameterName::IndexVector{i, j}),
+                    skip |= ValidateStructType(geometry_loc.dot(Field::geometry).dot(Field::triangles),
+                                               "VK_STRUCTURE_TYPE_ACCELERATION_STRUCTURE_GEOMETRY_TRIANGLES_DATA_KHR",
+                                               &(pInfos[i].pGeometries[j].geometry.triangles),
+                                               VK_STRUCTURE_TYPE_ACCELERATION_STRUCTURE_GEOMETRY_TRIANGLES_DATA_KHR, false,
+                                               kVUIDUndefined, "VUID-VkAccelerationStructureGeometryTrianglesDataKHR-sType-sType");
+                    skip |= ValidateStructPnext(geometry_loc.dot(Field::geometry).dot(Field::triangles), nullptr,
```

Comment:
Should it be:
```suggestion
                    skip |= ValidateStructPnext(geometry_loc.dot(Field::geometry).dot(Field::triangles).dot(Field::pNext), nullptr,
```
Or is it handled it `ValidateStructPnext`?